demo-notebooks/guided-demos/3_basic_interactive.ipynb

```diff
@@ -125,6 +128,19 @@
     "Now we can connect directly to our Ray cluster via the Ray python client:"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c9436436",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from codeflare_sdk import generate_cert\n",
+    "# Export the environment variables to enable TLS\n",
```

Comment:
I think this should be updated to reflect these functions more accurately. This is in 3 different places in the PR.
```suggestion
    "# Create required TLS cert and export the environment variables to enable TLS\n",
```